starter-code/index.html

```diff
@@ -1,31 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel="stylesheet" href="./css/main.css">
   <title>Instagram</title>
 </head>
-<body>
-  Instagram
+<body class ="css3gradient">
+<div class="divForPhones">
 
-  Capture and Share the World's Moments
+    <img src="/Users/ivanjorge/Prework/code/labs/lab-css-instagram-clone/starter-code/images/phones.png" alt ="" class="phones">
+</div>
+<div class = "combined">
+  <div class="instagram">
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+   <a href = "/Users/ivanjorge/Prework/code/labs/lab-css-instagram-clone/starter-code/index.html" ><img src="/Users/ivanjorge/Prework/code/labs/lab-css-instagram-clone/starter-code/images/brand.png" alt="" class ="picbrand"></a>
+  </div>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+  <button type="button" name="button" class="buttonlogin"> <img src="/Users/ivanjorge/Prework/code/labs/lab-css-instagram-clone/starter-code/images/home.png" alt="" class="picxhouse"><span class ="login">Log in</span></button>
+</div>
+<div class="insideTheWideImage">
 
-  Oh yeah, did we mention it's free?
+    <h2 class="header2">Capture and Share <br>the World's Moments</h2>
 
-  Download on the App Store
-  Get it on Google Play
+    <p class="paragraph1">Instagram is a <b>fast, beautiful</b> and <b>fun</b> way to share your life with friends and family.</p>
```

Comment:
For the sake of making your code easier to read, give your paragraph tag a class that describes what's inside it. For example, this one could be "ig-description-part1". Since all of the <p> tags have the same style, though, you'd be better off giving them the class "ig-description", since none of them are changing.